app/controllers/concerns/search_match_length_ab_testable.rb

```diff
@@ -0,0 +1,25 @@
+module SearchMatchLengthABTestable
```

Comment:
Thinking out loud here: I wonder if we should remove this module and move the logic into the controller? Unlike the navigation A/B test, the search A/B test only affects a single controller. And we might end up adding multiple A/B tests, which will probably push us to extract this code in a different way.